packages/app/src/files-manager/actions/upload-file.action.ts

```diff
@@ -89,6 +93,7 @@ export const uploadFile = createAsyncThunk(
         )
       )
     }
+    state = getState()
```

Comment:
I am wondering about this line, will this give something to re-read state one more time?